### DISCUSSION TITLE
Virus Total flags opensnitch_1.6.2-1_amd64.deb as unsafe

### DISCUSSION THREAD

**Discussion by TheCapsLock** [evilsocket/opensnitch#1085] [Q&A]
See the report here : https://www.virustotal.com/gui/file/edc9dad5f2e5fbb83c0cdc4a672d74ef3ab8cbfcc30ff1f9e26294cbc9ee0e04/detection

same for opensnitch_1.6.2-1_amd64.deb : https://www.virustotal.com/gui/file/edc9dad5f2e5fbb83c0cdc4a672d74ef3ab8cbfcc30ff1f9e26294cbc9ee0e04

**Comment by gustavo-iniguez-goya** [comment]
Hi @TheCapsLock ,

I've downloaded the package from our release section, and the md5 match my local copy and the gpg signature is correct.

There's no malicious activity reported and all the operations seems normal to me: https://www.virustotal.com/gui/file/edc9dad5f2e5fbb83c0cdc4a672d74ef3ab8cbfcc30ff1f9e26294cbc9ee0e04/behavior

Also the extra scan performed by a 3rd party service does not report any malicious behaviour/file: 
https://www.filescan.io/reports/edc9dad5f2e5fbb83c0cdc4a672d74ef3ab8cbfcc30ff1f9e26294cbc9ee0e04/01c9fab3-1f60-4596-8f91-10f37ff529fb/overview

So it looks like a false positive to me. I'll close this issue, but if you have more questions, please contact Ikarus about what exactly does a "Linux.Chia" malware, and why a .deb package is detected as malware. I haven't found info about it.

**Comment by TriMoon** [comment]
@TheCapsLock actually Virus Total is not flaging it as unsafe, it is `ikarus` who flags it, and that even doesn't mention it is "unsafe"...
If at all you should consider "Ikarus" as being an "unsafe" app itself :wink: 

"Chia" is just a blockchain system, which can hardly be interpreted as "malware" or "unsafe", which isn't even included in opensnitch AFAIK...
https://duckduckgo.com/?t=ffab&q=Linux.Chia&ia=web
